Tests/test_imagefont.py

```diff
@@ -750,6 +753,93 @@ def test_variation_set_by_axes(self):
         font.set_variation_by_axes([100])
         self._check_text(font, "Tests/images/variation_tiny_axes.png", 32.5)
 
+    @pytest.mark.parametrize(
+        "anchor",
+        (
+            # test horizontal anchors
+            "ls",
+            "ms",
+            "rs",
+            # test vertical anchors
+            "ma",
+            "mt",
+            "mm",
+            "mb",
+            "md",
+        ),
+    )
+    def test_anchor(self, anchor):
+        name, text = "quick", "Quick"
+        path = "Tests/images/test_anchor_%s_%s.png" % (name, anchor)
```

Comment:
```suggestion
        path = f"Tests/images/test_anchor_{name}_{anchor}.png"
```

Reply:
Ah yes, this was started before 3.5 support was dropped.